app/components/AR.js

```diff
@@ -73,6 +73,7 @@ const HTML = `
 
 const BRIDGE_INJECT_SCRIPT = `
   var targetLocIdx = 0;
```

Comment:
They're only called locs in the context of the webview that's overlayed on top of the camera, as a way to keep the labels separate across the webview bridge.